<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{\nbackground-color:rgb(170, 139, 159);
border: 1px solid gray;\n}\nQPushButton:pressed{\nbackground-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,\n										stop: 0, #dadbde, stop: 1 #f6f7fa);\n}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignRight';
border-bottom: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(246, 255, 216);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_to2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>to2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_to8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>to8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_to16">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>to16</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>68</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 105, 105);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_A">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_B">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_F">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(211, 211, 157);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2_to_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>68</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2to10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8_to_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>68</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8to10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton16_to_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>68</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(199, 217, 255);
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
										stop: 0, #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>16to10</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
